src/main.js

```diff
@@ -63,22 +64,22 @@ function render(arrayToRender) {
     for (var i = 0; i < arrayToRender.length; i++) {
       var imageSource = getImageSourceFromIdea(arrayToRender[i]);
         markup += `
-        <article class="idea" id="${i}">
+        <article class="idea" id="${arrayToRender[i].id}">
```

Comment:
I know you need the specific id in order to attach it to a comment, I recommend we use a data attribute. I used one in a previous project that I can show you and it worked really well. 
Here's a link https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes
We can add any extra info we need using data attributes, solving the ID problem.